<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>738</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>738</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>738</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PDF Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>extras/mail.png</normaloff>extras/mail.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>98</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>98</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>98</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Veranstaltung:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="configBtn">
       <property name="text">
        <string>EINSTELLUNGEN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutBtn">
       <property name="text">
        <string>Impressium</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="instructionsBtn">
       <property name="text">
        <string>Anleitungen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>416</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Wer soll eingeladet sein?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="G1_Ck">
         <property name="text">
          <string>G1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="G2_Ck">
         <property name="text">
          <string>G2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="G3_Ck">
         <property name="text">
          <string>G3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="G4_Ck">
         <property name="text">
          <string>G4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="G5_Ck">
         <property name="text">
          <string>G5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="G6_Ck">
         <property name="text">
          <string>G6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="G7_Ck">
         <property name="text">
          <string>G7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="G8_Ck">
         <property name="text">
          <string>G8</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="G9_Ck">
         <property name="text">
          <string>G9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QCheckBox" name="G10_Ck">
         <property name="text">
          <string>G10</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Wer unterschreibt das Dokument?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="toSign1">
         <property name="text">
          <string>Person 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="toSign2">
         <property name="text">
          <string>Person 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="toSign3">
         <property name="text">
          <string>Person 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="toSign4">
         <property name="text">
          <string>Person 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="toSign5">
         <property name="text">
          <string>Person 5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="eventBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>205</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>205</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>205</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>---------</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Interne Veranstaltung mit Zeitplan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Veranstaltung</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Info Brief</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Info Brief</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="viewTemplateBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dokumentvorlage</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="currConfigBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>409</width>
      <height>292</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>409</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Generate PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eventSetting">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Veranstaltung Einstellungen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
